.github/workflows/test_e2e.yml

```diff
@@ -9,10 +9,10 @@ on:
 
 jobs:
   e2e:
-    runs-on: ubuntu-latest
+    runs-on: self-hosted
```

Comment:
I want to make sure that we have clear documentation on how to change over hosting configuration in case we have to migrate hosts, and you aren't available.